test/integration/targets/handler_race/roles/random_sleep/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 # tasks file for random_sleep
 - name: Generate sleep time
   set_fact:
-    sleep_time: "{{ 60 | random }}"
+    sleep_time: "{{ 30 | random }}"
```

Comment:
This isn't related to any fix, but when running tests I noticed these tests were pretty slow.